src/picongpu/include/particles/ionization/ionizationEnergies.param

```diff
@@ -36,9 +36,23 @@ namespace SI
 
     /* ionization energy for ground state hydrogen: 13.6 eV --> converted to Joule */
     /* Joule = kg * m^2 / s^2 */
-    const double IONIZATION_ENERGY = 2.179e-18; 
-    /* ionization field strength (E-field) for -""-: in Volt / meter */
-    const double IONIZATION_EFIELD = 5.14e11;
+    PMACC_CONST_VECTOR(float_X,1,IONIZATION_ENERGY,
+        2.179e-18
+    );
+    /* ionization field strength (E-field) for -""-: in Volt / meter 
```

Comment:
it's a happy 6-armed creature ... just kidding, my bad :stuck_out_tongue: 
the name of an element from the PSE is supposed to be here